Doxyfile

```diff
@@ -1570,6 +1563,17 @@ TREEVIEW_WIDTH         = 250
 
 EXT_LINKS_IN_WINDOW    = NO
 
+# If the HTML_FORMULA_FORMAT option is set to svg, doxygen will use the pdf2svg
```

Comment:
SVG looks nicer, why don't we want to use the pdf2svg tool?

Reply:
I don't think we use formulas yet but SVG would definitely look nicer if we did. This is the default value that came from generating using the latest version of Doxygen. 

If we did use formulas and wanted SVG in that case we'd need to also install pdf2svg or inkscape on the machine as part of the doc generation because they're not available on the agents by default.